### PR TITLE
Fix/sections list

### LINKED ISSUE #171: [PROFILES] Improve sections list
## Description
We have new designs.

## Screenshots
screen shot 2018-07-10 at 01 36 54

## Files
- https://app.zeplin.io/project/5b1a4192c1a8233544fcc66a/screen/5b435f81cf2739b55d0ec56c

## Tasks
- [ ] Improve according to designs.

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
Improves the section list view according to design

Fixes #171 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
screenshot from 2018-07-10 13-34-09

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/templates/takwimu/_includes/profile/sections_list.html [hunk 1]
```diff
@@ -1,16 +1,41 @@
...
-      {% include 'takwimu/_includes/profile/country_selector.html' with current_page=page %}
+      <div class="section-list-country-head">
+          <img class="rounded-circle border-light mr-2"
+            src={% with 'img/flags/'|add:page.title|lower|add:'.svg' as image_static %}
+                  {% static image_static %}
+                {% endwith %}
+            border="8"
+            style="border-style:double; width: 68px; height: 67px;"/>
+          {{ page.title }}
+      </div>
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/profile/sections_list.html]: Did you remove the country selection? I'd prefer that we translate edits to the re-usable component.

File: takwimu/templates/takwimu/_includes/profile/sections_list_detail.html [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
+        <span class="text-light py-3 px-4" style="background-color: #1e4a7e; box-shadow: 0 0 4px 0 rgba(0, 0, 0, 0.26);"><i class="fa fa-eye mr-1"></i> YOU ARE HERE</span>
       </div>
 {% elif page.subpage_types|first == 'takwimu.ProfileSectionPage' %}
   <a href="{{ section.url|default:'#' }}">
-    <div class="jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
+    <div class=" container jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
 {% else %}
   <a href="{{ section.url|default:'../' }}">
-    <div class="jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
+      <div class="container jumbotron jumbotron-fluid border-top bg-white pt-4 pb-5 mb-0">
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/profile/sections_list_detail.html]: Was just about to write about this `.container` class. Well done @KhadijaMahanga.

File: takwimu/templates/takwimu/_includes/profile/sections_list.html (hunk 1 above)
Comment (KhadijaMahanga) [reply takwimu/templates/takwimu/_includes/profile/sections_list.html]: From the design it looked like its not a button and has no drop-down selection. Shouldn't that be the case?

**Comment by KhadijaMahanga** [comment]
@DavidLemayian I have pushed the changes requested

**Pull request merged**